src/store.js

```diff
@@ -0,0 +1,32 @@
+import { Record } from 'immutable'
+import { createStore, applyMiddleware } from 'redux'
+
+const initialState = new (Record({
+  workspaces: new (Record({
```

Comment:
I don't love this here. My thinking is that I would like it better if this were in the workspaces component file and added to the store at the end. See below.